pkg/components/storage.go

```diff
@@ -11,6 +13,17 @@ import (
 	"github.com/openshift/microshift/pkg/config/lvmd"
 )
 
+// getCSIPluginConfig searches for a user-defined lvmd configuration file in /etc/microshift.  If found, returns
+// the lvmd config.  If not found, returns a default-value lvmd config.  If an unmarshalling errors, returns nil
+// and the error.
+func getCSIPluginConfig() (*lvmd.Lvmd, error) {
+	lvmdConfig := filepath.Join(filepath.Dir(config.DefaultGlobalConfigFile), lvmd.LvmdConfigFileName)
+	if _, err := os.Stat(lvmdConfig); errors.Is(err, os.ErrNotExist) {
+		return lvmd.NewLvmdConfigFromFile(filepath.Join(lvmdConfig, lvmd.LvmdConfigFileName))
```

Comment:
`lvmdConfig` already points to `/etc/microshift/lvmd.yaml`

```suggestion
		return lvmd.NewLvmdConfigFromFile(lvmdConfig)
```